src/applications/proxy-rewrite/redirects/crossDomainRedirects.json

```diff
@@ -1333,5 +1333,15 @@
     "domain": "www.benefits.va.gov",
     "src": "/gibill/yellow_ribbon/2019/states/wy.asp",
     "dest": "/education/yellow-ribbon-participating-schools/"
+  },
+  {
+    "domain": "www.vba.va.gov",
+    "src": "/pubs/forms/VBA-21-0958-ARE.pdf",
+    "dest": "/decision-reviews/legacy-appeals/"
+  },
```

Comment:
Hm our JS snippet won't execute on a static file like this PDF, so this won't work. The VBA crew will have to do it

Reply:
Who do we contact on the VBA for that?